static/app/utils/discover/fieldRenderers.tsx

```diff
@@ -783,12 +791,24 @@ const SPECIAL_FUNCTIONS: SpecialFunctions = {
     const userMiseryField = fieldName;
 
     if (!(userMiseryField in data)) {
-      return <NumberContainer>{emptyValue}</NumberContainer>;
+      return (
+        <Tooltip title={missingUserMisery} showUnderline isHoverable>
+          <RectangleRelativeOpsBreakdown>
+            <NumberContainer>{emptyValue}</NumberContainer>
+          </RectangleRelativeOpsBreakdown>
```

Comment:
I find the name of `RectangleRelativeOpsBreakdown` to be a bit unintuitive. It looks like all it does is set the position and width. Perhaps we rename this component so it also suits this use case?

Reply:
I don't think this is needed, my bad pushing this branch while still experimenting a bit.